### PR TITLE
feat/WIT-101-trivia-command

### PR DESCRIPTION
**Pull request by joycceh**
- A slash command called trivia-command 
- When used it will generate a random trivia question to the Discord channel 
- Features implemented includes ensuring users can only attempt each trivia question once

### CODE REVIEW

File: cogs/askTriviaQuestions.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+            choices,
+            answerIndx,
+            questionTxt,
+            questionDiff,
+            self.incorrect_answers,
+            self.answered_users,
+            False,
+        )
+        await inter.followup.send(
+            # content=f"Trivia Question!\n\n",
```
Comment (aolinxuu) [inline cogs/askTriviaQuestions.py:77]: delete comment

File: cogs/askTriviaQuestions.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        if selected_choice == self.correct_choice:
+            # if the selected choice is correct, display a text
+            await self.updateMessageCorrect(interaction)
+            # await interaction.response.send_message(f"You selected the correct choice!")
+
+        else:
+            self.incorrect_answers += 1
+            # if the selected choice is incorrect, do nothing
+            await self.updateMessageIncorrect(interaction)
+            # await interaction.response.send_message(f"You selected the wrong choice!")
```
Comment (aolinxuu) [inline cogs/askTriviaQuestions.py:160]: delete comment

File: cogs/askTriviaQuestions.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        self.incorrect_answers = 0
+        self.answered_users = set()
+
+    @app_commands.command(
+        name="ask-trivia-questions", description="Asks trivia questions!"
+    )
+    @app_commands.describe(difficulty="choose a difficulty")
+    @app_commands.choices(
+        difficulty=[
+            app_commands.Choice(name="EASY", value=1),
```
Comment (aolinxuu) [inline cogs/askTriviaQuestions.py:20]: change to lower case (Easy, Medium, Hard)

**Review by aolinxuu — COMMENTED** [review]
The functionality is great! However there are style inconsistencies. I noticed some variable names were camel case (eg camelCase) and some are snake case (eg snake_case). Choose one to stick with. Be careful not the change the ones that are library functions.

**Comment by aolinxuu** [comment]
Commit names are mainly good.. expect a few which are mine. I will need to remember to add the description of the commit.

**Review by katejoh — APPROVED** [review]
Great work!

**Pull request merged**